hms-api/src/handler/ideaComment/create.ts

```diff
@@ -7,10 +7,9 @@ import IdeaCommentCreateResponse from '../../rest/ideaComment/IdeaCommentCreateR
 export async function create(event, context, callback) {
   await wrapHandler(async () => {
     const request = IdeaCommentCreateRequest.parse(event.body);
-    const ideaId = event.pathParameters.id;
     const ideaComment = await createIdeaComment(
       request.userId,
-      ideaId,
```

Comment:
idk about this 🤔

Reply:
im sure there was a good reason why i did this 
maybe it was just for consistency with all the other create methods

Reply:
well, we SHOULD be pulling it from the path and not from the request

Reply:
or, we should remove it from the path (means changing the frontend too) and only get it from the request

Reply:
if this dont make the tests impossible then i can try it